samples/fbuffer/Connectors/SumConnector.cpp

```diff
@@ -163,7 +162,7 @@ long long int SumConnector::REDISListLen(const std::string &sum_name) noexcept {
     }
     else {
         DARWIN_LOG_ERROR("SumConnector::REDISListLen:: Error while querying key '" + sum_name + "'");
-        result = 0.0L;
+        return -1;
```

Comment:
This modification seems to be done to resolve an issue that I don't know nor can identify the issue, I can't review it

Reply:
the previous return didn't specify an error in the process, now it does (and is checked in BufferThread.cpp:34)